packages/admin-next/dashboard/src/routes/orders/order-detail/components/order-payment-section/order-payment-section.tsx

```diff
@@ -4,45 +4,62 @@ import {
   Refund as MedusaRefund,
   Order,
 } from "@medusajs/medusa"
+import { HttpTypes } from "@medusajs/types"
 import {
   Badge,
   Button,
   Container,
   Heading,
   StatusBadge,
   Text,
+  toast,
   Tooltip,
+  usePrompt,
 } from "@medusajs/ui"
 import { format } from "date-fns"
 import { useTranslation } from "react-i18next"
 import { ActionMenu } from "../../../../../components/common/action-menu"
+import { useCapturePayment } from "../../../../../hooks/api"
+import { formatCurrency } from "../../../../../lib/format-currency"
 import {
   getLocaleAmount,
   getStylizedAmount,
 } from "../../../../../lib/money-amount-helpers"
 
 type OrderPaymentSectionProps = {
-  order: Order
+  order: HttpTypes.AdminOrder
 }
 
 export const OrderPaymentSection = ({ order }: OrderPaymentSectionProps) => {
+  const payments = order.payment_collections
+    .map((collection) => collection.payments)
+    .flat(1)
+    .filter(Boolean)
+
+  const refunds = payments
+    .map((payment) => payment?.refunds)
+    .flat(1)
+    .filter(Boolean)
+
   return (
     <Container className="divide-y divide-dashed p-0">
       <Header order={order} />
       <PaymentBreakdown
-        payments={order.payments}
-        refunds={order.refunds}
+        payments={payments}
+        refunds={refunds}
         currencyCode={order.currency_code}
       />
-      <Total payments={order.payments} currencyCode={order.currency_code} />
+      <Total payments={payments} currencyCode={order.currency_code} />
     </Container>
   )
 }
 
 const Header = ({ order }: { order: Order }) => {
   const { t } = useTranslation()
 
-  const hasCapturedPayment = order.payments.some((p) => !!p.captured_at)
+  const hasCapturedPayment = order.payment_collections.some(
+    (p) => !!p.captured_at
```

Comment:
seems that `PaymentCollection` doesn't have this field ATM since we haven't define what it means for a PC to be captured because it can have multiple payments and refunds

Reply:
good catch, removed this based on our discussions.